### PR TITLE
Feature/implementjs

### PR DESCRIPTION
**Pull request by bhskr44**
For this milestone of your portfolio website, you will implement the mobile menu using your recently acquired JavaScript knowledge.

- [x] For this project, you should add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

In mobile, implement the following features:

- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.   
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  } else {
+    menuButton.style.display = 'block';
+    menuIcon.className = 'fa fa-close fa-2x';
+    menuIcon.style.color = 'white';
+    navId.classList.add('add-bg-to-nav');
+    mainElement.classList.add('add-blur');
+    logoText.classList.add('add-blur');
+  }
+}
+/* eslint-disable */
```
Comment (GzimAsani) [inline js/main.js]: - [ ] The hamburger button initial appears as it should, and when clicked it sends you to the correct part of the page. But after that the hamburger button disappears from dom, kindly check your function again and the errors that is producing
image

File: index.html [hunk 2]
```diff
@@ -25,23 +25,32 @@
...
           </div>
-          <div class="button">
-            <i class="fa fa-bars fa-2x"></i>
+          <div
+            class="button"
+            id="mobile-menu"
+            class="mobile-menu-class"
+            onClick="toggleMenu();"
+          >
+            <i id="hm-menuIcon" class="fa fa-bars fa-2x"></i>
```
Comment (GzimAsani) [inline index.html]: - [ ] Update the weight/thickness of the X, in order to match the design
Yours: 
image

Design: 
image

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const menuButton = document.getElementById('destop-menu');
+const mainElement = document.getElementById('section1');
+const navId = document.getElementById('navId');
+const menuIcon = document.getElementById('hm-menuIcon');
+const logoText = document.getElementById('logo-text');
+const mobileMenu = document.getElementById('mobile-menu');
+// const mobileMenu = document.getElementById('mobile-menu');
```
Comment (GzimAsani) [inline js/main.js]: - [ ] Remove commented out code if you do not plan to use them

File: js/main.js (hunk 1 above)
Comment (GzimAsani) [inline js/main.js]: - [x] If possible do not disable linters because they are there to help you write better code

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @bhskr44      :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your hamburger button, it correctly sends to the selected part of page:+1:  
 
 ## Required Changes 

- [ ] In your readme file kindly include how to clone your project, be as specific as you can.
- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Bhaskar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**